src/node/blockstorage.cpp

```diff
@@ -387,7 +387,13 @@ bool BlockManager::LoadBlockIndex(const std::optional<uint256>& snapshot_blockha
     }
 
     if (snapshot_blockhash) {
-        const AssumeutxoData au_data = *Assert(GetParams().AssumeutxoForBlockhash(*snapshot_blockhash));
+        const auto& maybe_au_data = GetParams().AssumeutxoForBlockhash(*snapshot_blockhash);
```

Comment:
`std::optional<AssumeutxoData> maybe_au_data`

Reply:
Ok, let me change it, I solved it that way because I saw how it's managed in `src/validation.cpp`:
https://github.com/bitcoin/bitcoin/blob/d53400e75e2a4573229dba7f1a0da88eb936811c/src/validation.cpp#L5359-L5368

Reply:
done!